_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title: Clear the SMMSTORE ROM
+description: >
+  How to fix the failed with exit status: exit code: 5 message
+keywords:
+  - Firmware
+  - Open Firmware
+  - SMMSTORE
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
```

Comment:
Let's add a Things you will need section. 

E.g. 1-2 flash drives, Phillips Screwdriver, 30mins. etc.